2020/10/2020-10-23-RIAA.md

```diff
@@ -4,115 +4,27 @@ GitHub
-source code being hosted on GitHub. Based on our review of the representative sample noted
-above, we have a good faith belief that most of the youtube-dl forks are infringing to the same
-extent as the parent repository.
+Most of this comes in the form of taking down anyone we don't like! It's great!
 
-The clear purpose of this source code is to (i) circumvent the technological protection measures
-used by authorized streaming services such as YouTube, and (ii) reproduce and distribute music
-videos and sound recordings owned by our member companies without authorization for such
-use. We note that the source code is described on GitHub as “a command-line program to
-download videos from YouTube.com and a few more sites.”1
+Because we're such morons, we believe that youtube-dl is infringing on our copyright with code we don't own, and again, because we've got you by the balls, you need to take it down.
 
-We also note that the source
-code prominently includes as sample uses of the source code the downloading of copies of our
-members’ copyrighted sound recordings and music videos, as noted in Exhibit A hereto. For
-example, as shown on Exhibit A, the source code expressly suggests its use to copy and/or
-distribute the following copyrighted works owned by our member companies:
+... What's a "streisand effect"?
 
-• Icona Pop – I Love It (feat. Charli XCX) [Official Video], owned by Warner Music Group  
-• Justin Timberlake – Tunnel Vision (Explicit), owned by Sony Music Group  
-• Taylor Swift – Shake it Off, owned/exclusively licensed by Universal Music Group
+We believe that the code is ONLY used to download the shit we own, and it has literally no other use at all. Really, it doesn't, you only use it to download our music videos.
 
-The source code notes that the Icona Pop work identified above is under the YouTube Standard
-license, which expressly restricts access to copyrighted works only for streaming on YouTube
-and prohibits their further reproduction or distribution without consent of the copyright owner;
-that the Justin Timberlake work identified above is under an additional age protection
-identifier; and that the request for the Taylor Swift work identified above is to obtain, without
-authorization of the copyright owner or YouTube, an M4A audio file from the audiovisual work
-in question.
+What do you mean it works the same as the browser does for decrypting the DRM?
 
-We have a good faith belief that this activity is not authorized by the copyright owner, its agent,
-or the law. We assert that the information in this notification is accurate, based upon the data
-available to us.
+We have a good faith that we're not complete and utter idiots, (even though we clearly are), and that youtube-dl infringes on literally all of our copyrights because it **CAN** be used to download our shit, and we refuse to recognize any other use of the code, because there are none.
```

Comment:
"We ***THINK*** that this activity is not authorized by the copyright owner, its agent, or the law. We assert that the information in this notification is accurate, based upon the data available to us. 
We have not actually asked the copyright owner, we think therefore it must be true"